metrics/github/repos.py

```diff
@@ -14,6 +14,9 @@ class Repo:
     is_archived: bool = False
     has_vulnerability_alerts_enabled: bool = False
 
+    def is_tech_owned(self):
+        return self.team in _TECH_TEAMS
```

Comment:
This method feels like it should be a property.

Reply:
Yeah, I wondered about that. I'm not sure what the criteria are for this decision. Should all zero-arg, side-effect-free methods be properties? I'll change it.